backend/api/product.controller.js

```diff
@@ -78,4 +78,28 @@ export default class ProductController {
             return false
         }
     }
+
+
+    static async getProductById(req, res, next) {
+        try {
+            const productId = req.params.id
+
+            console.log(productId)
+
+            let product = await ProductDAO.findProductById(productId)
+
+            if (product) {
+/*                 res.json({
+                    _id: product._id,
```

Comment:
I recommend deleting this portion of code that is commented out.

Reply:
Thank you for your recommendation. I have deleted this commented code.